tests/create_task_ignore_error.yml

```diff
@@ -0,0 +1,52 @@
+name: Create Tes Task Job With Ignore Error Flag
+description: Job to create a new TES Task with ignore_error flag
+service: TES
+versions:
+  - 1.1.0
+tags:
+  - logical
+  - create task
+  - create task ignore error test
+  - all
+jobs:
+  - name: create_task
+    description: Create a new TES task
+    endpoint: /tasks
+    operation: POST
+    request_body: |
+      {
+        "name": "CompTest",
+        "description": "CompTest",
+        "executors": [
+          {
+            "image": "alpine",
+            "command": [
+              "ERROR"
+            ],
+            "ignore_error": true
```

Comment:
Should we add this test after the `filter` PR? As discussed, we can implement the array size check for the executor logs.

Reply:
Yes, I think that makes sense